.eslintrc.yml

```diff
@@ -6,6 +6,7 @@ env:
   browser: true
   mocha: true
   node: true
+  es6: true
 
 parserOptions:
   ecmaVersion: 6
```

Comment:
`es6: true` makes `ecmaVersion: 6` unnecessary (see https://github.com/eslint/eslint/pull/7067). Do you mind removing `parserOptions`?